### DISCUSSION TITLE
Can't upload to s3

### DISCUSSION THREAD

**Discussion by leonidmamontov27** [Altinity/clickhouse-backup#193] [Q&A] [answered]
2021/05/06 07:06:43 error can't upload: InvalidArgument: 
	status code: 400, request id: 4S3WH1HASRJH6NYG, host id: 77qAZ3lHUKQZrBBPXmoOyepckdq8YszoWj875iCJB4ae2oq21O6cf4cEQ/UA+oQME/gFrs4tNjA=

**Comment by Slach** [comment, marked as answer]
not enough information
try to use latest 1.4.7 version
